packages/expo-module-scripts/eslint.config.base.js

```diff
@@ -0,0 +1,58 @@
+const universeNativeConfig = require('eslint-config-universe/flat/native');
+const universeNodeConfig = require('eslint-config-universe/flat/node');
+
+const universeRoot = require.resolve('eslint-config-universe/package.json');
+const { defineConfig } = require(
+  require.resolve('eslint/config', {
+    paths: [universeRoot],
+  })
+);
+const globals = require(require.resolve('globals', { paths: [universeRoot] }));
+
+module.exports = defineConfig([
+  {
+    extends: universeNativeConfig,
+
+    rules: {
+      'no-restricted-imports': [
+        'warn',
+        {
+          // fbjs is a Facebook-internal package not intended to be a public API
+          patterns: ['fbjs/*', 'fbjs'],
+        },
+      ],
+    },
+  },
+  {
+    files: ['**/__tests__/*'],
+
+    languageOptions: {
+      globals: {
+        ...globals.node,
+        __DEV__: true,
+      },
+    },
+  },
+  {
+    files: ['./*.config.js', './.*rc.js'],
+    extends: universeNodeConfig,
+  },
+  {
+    files: ['**/*.ts', '**/*.tsx', '**/*.d.ts'],
+
+    rules: {
+      // NOTE: This is already handled by TypeScript itself
+      // Turning this on blocks legitimate type overloads
+      // TODO(@kitten): Please move this to universe
+      'no-redeclare': 'off',
+      '@typescript-eslint/no-redeclare': 'off',
+
+      // NOTE: Handled by TypeScript
+      // TODO(@kitten): Please move this to universe
+      'no-unused-expressions': 'off',
+      'no-unused-vars': 'off',
+      '@typescript-eslint/no-unused-expressions': 'off',
+      '@typescript-eslint/no-unused-vars': 'off',
+    },
+  },
```

Comment:
👀  I've not seen these comments before but noted. I can look at keeping those rules only for non-ts files.

Reply:
This might be contentious :smile: 
Basically, they may or may not make sense in userland. Mainly, in our own repo having this be handled by TypeScript gives us "more complete errors" and TypeScript has these two behaviours built-in already.
But in userland, the advantage of using the ESLint rules over the TypeScript `unused` errors is that we can warn instead, for example